src/aws_encryption_sdk/structures.py

```diff
@@ -152,3 +152,41 @@ class MessageHeader(object):
     content_aad_length = attr.ib(hash=True, validator=instance_of(six.integer_types))
     header_iv_length = attr.ib(hash=True, validator=instance_of(six.integer_types))
     frame_length = attr.ib(hash=True, validator=instance_of(six.integer_types))
+
+
+@attr.s
+class CryptoResult(object):
```

Comment:
Any value in calling this AwsCryptoResult to match Java?

Reply:
IMO no, but I'm willing to be convinced otherwise.

My reasoning is that aside from the name/branding and the AWS KMS keyring, nothing about the ESDK is AWS-specific. As such, I've tried to avoid including "AWS" in the names of things except where they specifically relate to AWS (ex: `keyrings.aws_kms`).

Reply:
Yeah, I'm fine with keeping it. As you know we changed the name more out of necessity than because it was a better name.